Documentation/kubernetes.md

```diff
@@ -40,9 +40,29 @@ kubectl create -f rook-operator.yaml
 kubectl get pod
 ```
 
-Now that the rook-operator pod is running, we can create the Rook cluster. See the documentation on [configuring the cluster](cluster-tpr.md).
-```bash
-kubectl create -f rook-cluster.yaml
+Now that the rook-operator pod is running, we can create the Rook cluster. For the cluster to survive reboots, 
+make sure you set the `dataDirHostPath` property. For more settings, see the documentation on [configuring the cluster](cluster-crd.md). 
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook
+---
+apiVersion: rook.io/v1alpha1
+kind: Cluster
+metadata:
+  name: rook
+  namespace: rook
+spec:
+  versionTag: master
+  dataDirHostPath:
```

Comment:
Had some discussion on that in slack, we aren't setting the default to keep it simpler until we implement the k8s local storage